app/src/main/java/com/kickstarter/viewmodels/EnvironmentalCommitmentsViewHolderViewModel.kt

```diff
@@ -48,7 +51,12 @@ class EnvironmentalCommitmentsViewHolderViewModel {
                 }
                 .filter { ObjectUtils.isNotNull(it) }
                 .map { requireNotNull(it) }
-                .subscribe(this.category)
+                .subscribe { this.category.onNext(it) }
+                .addToDisposable(disposables)
+        }
+
+        override fun clearDisposables() {
+            disposables.clear()
```

Comment:
this doesn't necessarily need to be part of the interface.  if you want you can clear these on the `onCleared` call from the viewmodel (you can override it)

Reply:
Okay that works. I was calling this from the view holder but I'll override the onCleared method instead.